types/redux-sentry-middleware/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for redux-sentry-middleware 0.0
+// Project: https://github.com/vidit-sh/redux-sentry-middleware#readme
+// Definitions by: Jan Dolezel <https://github.com/dolezel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { Action, Middleware } from "redux";
+import * as Sentry from "@sentry/browser";
+
+export interface SentryMiddlewareOptions<T> {
+    breadcrumbDataFromAction?: (action: Action) => any;
+    actionTransformer?: (action: Action) => Action;
+    stateTransformer?: (state: T) => T;
+    breadcrumbCategory?: string;
+    filterBreadcrumbActions?: (action: Action) => boolean;
+    getUserContext?: (state: T) => Sentry.User;
+    getTags?: (state: T) => Sentry.SentryEvent['tags'];
+}
+
+export default function createSentryMiddleware(sentry: typeof Sentry, options?: SentryMiddlewareOptions<any>): Middleware;
```

Comment:
The `any` generic argument is odd here. I would suggest either making createSentryMiddleware a generic function and passing the T from there into the options, or removing the generic argument from the options interface (and make the individual functions on the interface be generic themselves).